src/Dialogs/Device/PortDataField.cpp

```diff
@@ -115,10 +115,8 @@ FillDefaultSerialPorts(DataFieldEnum &df) noexcept
   for (unsigned i = 1; i <= 10; ++i) {
     TCHAR buffer[64];
     _stprintf(buffer, _T("COM%u"), i);
-    if (DosDeviceExists(buffer)) {
-      _tcscat(buffer, _T(":"));  // add the colon to the path string
```

Comment:
I don't understand why this gets removed now, when you just added this line of code in the preceding commit.